ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -2296,3 +2299,19 @@ def test065_download_suborder_sheet_for_single_order(self):
                                   f"{str(fixed_sheet_row_data)} : {str(formatted_orders[index])}")
             for item in formatted_orders[index]:
                 self.assertIn(item, fixed_sheet_row_data)
+
+    def test066_search_with_test_unit_name(self):
+        """
+
+        LIMS-6664 Orders:Test unit search approach
+        allow user to search with test unit name in the drop down list of the order form
+        """
+        self.test_units_page.get_test_units_page()
+        self.test_units_page.open_configurations()
+        self.test_units_page.open_testunit_name_configurations_options()
+        old_values = self.test_units_page.select_option_to_view_search_with(view_search_options=['Name'])
+        self.info('go to orders page')
+        self.order_page.get_orders_page()
+        fields = self.order_page.split_test_unit_list()
```

Comment:
At this function you set material type of suborder to random material type, So if this selected material has no test units with it . This test case will pass as 'fields = (0,0)'.
so, we can select random test unit and set material type to its material.
Or, @0xIslamTaha  Can we set it to "Raw Material" directly ?